radio/src/lua/lua_api.h

```diff
@@ -41,6 +41,11 @@ extern "C" {
   #endif
 #endif
 
+#if !defined(CLI)
+#define LUA_FIFO_SIZE 256
+extern Fifo<uint8_t, LUA_FIFO_SIZE> luaRxFifo;
```

Comment:
256 bytes RAM
we take the risk that some scripts which are OK in 2.3.5 are KO in 2.3.6

Reply:
this buffer could be dynamically created on first `serialRead`

Reply:
Agreed. Buffer is now dynamically created. This should also help with the ram compatibility, right? (Since older scripts do not call serialRead and therefore do not instantiate the fifo?)

Reply:
exact